numpy/core/fromnumeric.py

```diff
@@ -1615,16 +1615,16 @@ def nonzero(a):
 
     Examples
     --------
-    >>> x = np.array([[1,0,0], [0,2,0], [1,1,0]])
+    >>> x = np.array([[3, 0, 0], [0, 4, 0], [5, 6, 0]])
```

Comment:
Changing these valeus makes them not be the same as the indices, which made things hard to follow